execution/inc/leatherman/execution/execution.hpp

```diff
@@ -187,6 +187,16 @@ namespace leatherman { namespace execution {
     };
 
     /**
+     * Encapsulates return value from executing a process: whether or not the command succeeded, output from stdout, output from stderr (if not redirected), and the exit code.
```

Comment:
Could we move the documentation for the members out of this and into the individual members?